apimanagementapi/ansible.yaml

```diff
@@ -0,0 +1,14 @@
+--- !ruby/object:Provider::Ansible::Config
+manifest: !ruby/object:Provider::Ansible::Manifest
+ metadata_version: '1.1'
+  status:
+    - preview
+  supported_by: 'community'
+  requirements:
+    - python >= 2.6
+    - requests >= 2.18.4
+  version_added: '2.9'
+  author: Junyi Yi (@JunyiYi)

```

Comment:
I think the author here can be replaced by something like <Type your name and GitHub handle here>. What do you think?